### DISCUSSION TITLE
Where's the Todo List stuff in the UI?

### DISCUSSION THREAD

**Discussion by Tony20221** [jasontaylordev/CleanArchitecture#780] [Q&A] [answered]
I see code that deals with ToDo Lists. Where's the TodoList stuff in the UI? I ran the WebUI project.

**Comment by hsandhar** [comment, marked as answer]
@Tony20221 

- You need to be logged in as a user/admin in order to **access** the Todo page.
- You should still be able to view the Todo in the nav menu 
- When you navigate to that page, it should redirect you to the Login page since this path uses AuthGuard.
- You can either register as a new user, or use the default admin account that is part of the dbContextInit --> `src/Infrastructure/Persistence/ApplicationDbContextInitialiser.cs `--> **username**: `administrator@localhost` **password**: `Administrator1!`
- Adding a few screenshots below for further context; hope it helps.

![image](https://user-images.githubusercontent.com/1728547/222973183-a65fdd0f-c56d-4611-8282-62323681f55b.png)

![image](https://user-images.githubusercontent.com/1728547/222973200-2df4a386-69a2-4b73-acab-60e44368881e.png)

![image](https://user-images.githubusercontent.com/1728547/222973216-3b4213c9-3dc8-49af-86a7-9cc95be3ab3f.png)

![image](https://user-images.githubusercontent.com/1728547/222973231-f283387e-017d-4300-b872-9ce96e64ab7b.png)

![image](https://user-images.githubusercontent.com/1728547/222973241-8564482c-d7c9-44a3-871a-508ecba52b56.png)

**Reply by Tony20221** [reply]
Thanks.  Todo, JWT & API menu options were missing in my nav-menu.component.html. I downloaded the template again and now they show up.
